.github/workflows/haskell-ci.yml

```diff
@@ -15,32 +15,31 @@ jobs:
           - ubuntu-latest
           - macOS-latest
           - windows-latest
-        cabal: [3.6]
+        cabal: [3.10]
         ghc:
-          - 8.8.4
           - 8.10.7
```

Comment:
Should we still officially support and CI test 8.10.7? I'm not against it, it's just that I believe we've said we'd only support the 3 most recent versions. I'm fine with still supporting at least the last version of GHC 8.